android/libraries/rib-base/src/main/java/com/badoo/ribs/android/recyclerview/RecyclerViewHost.kt

```diff
@@ -19,6 +19,7 @@ interface RecyclerViewHost : Rib {
         fun initialElements(): List<T>
         fun recyclerViewHostInput(): ObservableSource<Input<T>>
         fun resolver(): RecyclerViewRibResolver<T>
+        fun recyclerViewFactory(): RecyclerViewFactory
         fun layoutManagerFactory(): LayoutManagerFactory
```

Comment:
If we allow to create recycler, should we still pass LayoutManager?

Reply:
Right now enforces not forgetting it by accident.